extension/helper/config.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param {ExtensionConfig} config
+ */
+function getBaseUrl(config) {
```

Comment:
eslint, need space before `(`, thought the automatic tests should be catching this